projects/views.py

```diff
@@ -195,10 +175,9 @@ def view_project(request, project_id):
             if isinstance(a.end, basestring):
                 a.end = datetime.strptime(a.end, '%Y-%m-%dT%H:%M:%SZ')
 
-            days_left = (a.end - datetime.today()).days
-            if days_left >= 0 and days_left <= 90:
+            if a.days_left >= 0 and a.days_left <= 90:
                 a.up_for_renewal = True
```

Comment:
This also looks like a derived property :)

```python
RENEWAL_START_WINDOW = 90

@property
def up_for_renewal(self):
    days_left = self.days_left
    return days_left >= 0 and days_left <= RENEWAL_START_WINDOW
```

And in that case, no need to deal with this `renewal_days`, just use `days_left` - I don't know if another property helps, given it always has the same value.